.prettierignore

```diff
@@ -0,0 +1,4 @@
+package.json
+package-lock.json
+yarn.lock
+dist
```

Comment:
Could we add `coverage` and maybe look over .gitignore for other directories to skip?

Reply:
I am considering adding the following:
`
coverage
node_modules
.vscode/*
/.angular/cache
`